tfx_addons/copy_example_gen/README.md

```diff
@@ -10,31 +10,71 @@
 **Project name:** CopyExampleGen component
 
 ## Project Description
-CopyExampleGen will allow the user to copy a pre-existing Tfrecord dataset or raw data and ingest it into the pipeline, ultimately skipping the process of shuffling and running the Beam job. This process will require a dict input with split_names and their respective URI. This will output an Examples Artifact (same as the Artifact output from the ExampleGen component)  in which downstream components can use.
+CopyExampleGen will allow the user to copy pre-existing tfrecords and ingest it into the pipeline as examples, ultimately skipping the process of shuffling and running the Beam job that is in the standard component, ExampleGen. This process will require a dict input with split names as keys and their respective URIs as the value from the user. Following suit, the component will set the artifact’s properties, generate output dict, and register contexts and execution. Lastly, it will output an Examples Artifact in which downstream components can use.
+
+Example of pipeline component definition:
```

Comment:
Done.

Reply:
The example should clearly show:
-Declaring the dict.
-Specifying the component (done).
-Anything that needs to be done with the json library (mentioned but command isn't shown).

Reply:
You should emphasize that the intention is to use with with tfrecord examples that have come out of an ExampleGen component previously (though you could mention it may work with other tfrecord files).